installing/installing_openstack/deploying-openstack-with-rootVolumes-etcd-on-local-disk.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="deploying-openstack-on-local-disk"]
+= Deploying OpenStack with rootVolumes and etcd on local disk
+include::_attributes/common-attributes.adoc[]
+:context: deploying-openstack-on-local-disk
+
+toc::[]
+
+include::snippets/technology-preview.adoc[]
+
+As a day 2 operation, you can deploy {rh-openstack-first} with rootVolumes and etcd on a local disk.
```

Comment:
I'm not sure how you can change the wording but one can read it as the root volume on local disk as well.

The use case is really:
* The control plane VMs store their data on root volumes (Cinder) except for etcd which runs on a local ephemeral disk.